src/main/java/edu/uw/edm/edmzuulproxy/healthcheck/GwsHealthIndicator.java

```diff
@@ -0,0 +1,39 @@
+package edu.uw.edm.edmzuulproxy.healthcheck;
+
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.stereotype.Component;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+import edu.uw.edm.gws.GroupsWebServiceClient;
+import edu.uw.edm.gws.model.GWSSearchType;
+import edu.uw.edm.gws.model.GroupReference;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Component
+@Slf4j
+public class GwsHealthIndicator implements HealthIndicator {
+
+    private GroupsWebServiceClient groupsWebServiceClient;
+
+    @Autowired
+    public GwsHealthIndicator(GroupsWebServiceClient groupsWebServiceClient) {
+        this.groupsWebServiceClient = groupsWebServiceClient;
+    }
+
+    @Override
+    public Health health() {
```

Comment:
@jfshenuw would you add a unit test to confirm that an exception from `groupsWebServiceClient.getGroupsForUser(username, GWSSearchType.direct);`  returns `Health.down` and any response returns `Health.up`

Reply:
added unit test with two test cases, one for "UP" and one for "DOWN"